services/mariadb/config/init/mariadb

```diff
@@ -1,9 +1,16 @@
-if [ ! "$(ls -A /data)" ]; then
+if [ ! "$(ls -A /data/aria_log_control)" ]; then
```

Comment:
With your new `debian` image, `/data` folder is no longer empty (you create `/data/log` folder in init loop), so the script was never executing mysql provision stuff.